httpx/_utils.py

```diff
@@ -134,9 +134,12 @@ def netrc_info(self) -> typing.Optional[netrc.netrc]:
             self._netrc_info = None
             for file_path in self.netrc_files:
                 expanded_path = Path(file_path).expanduser()
-                if expanded_path.is_file():
-                    self._netrc_info = netrc.netrc(str(expanded_path))
-                    break
+                try:
+                    if expanded_path.is_file():
+                        self._netrc_info = netrc.netrc(str(expanded_path))
+                        break
+                expect PermissionError:
+                    pass
```

Comment:
Skipping sounds reasonable, but should we at least log a debug message about this error?

(You can use `get_logger()` to instantiate a logger in this module, if there isn’t one yet. Might need to revise the order of functions if logging utils are later down this file.)